package.json

```diff
@@ -5,22 +5,31 @@
     "typings": "./dist/typings/index.d.ts",
     "main": "dist/commonjs/index.js",
     "types": "dist/typings/index.ts",
+    "bin": {
+        "fusebox": "bin/fsbx"
+    },
     "directories": {
+        "bin": "bin",
         "test": "test",
         "modules": "modules",
         "lib": "dist/commonjs",
-        "typings": "dist/typings"
+        "typings": "dist/typings",
+        "docs": "docs"
     },
     "files": [
+        "bin",
         "src",
         "dist",
         "modules",
         "test",
         "docs"
     ],
     "scripts": {
+        "test-cli": "fusebox",
         "help": "node bin/fsbx --help",
-        "test": "TRAVIS=true ./node_modules/mocha/bin/mocha && node ./test.js",
+        "lint-fix": "eslint src/*.ts src/**/*.ts src/**/**/*.ts test/*.js test/**/*.js --fix",
```

Comment:
there are a *lot* of eslint plugins that tslint does not have. It is not using default eslint parser, I've already set it up in master https://github.com/fuse-box/fuse-box/blob/master/.eslintrc.js#L2 

just the script was removed because there were some files that were autofixed in /modules/ that were fixed perfectly valid, however, they are included in any build chain so they are not compiled to es3 which causes some (very minor) accessibility issues.

However, modules is no longer linted, after the initial fix, the code base has a uniform style.